samples/drivers/lora/send/README.rst

```diff
@@ -0,0 +1,36 @@
+.. zephyr:code-sample:: lora
+   :name: lora send
+   :relevant-api: lora_radio
+
+   Transmit a preconfigured payload every second using the lora radio.
+
+Overview
+********
+
+This sample demonstrates how to use the lora radio driver to configure
+the encoding settings and send data over the radio.
+
+Transmitted messages can be received by building and flashing the accompanying
+lora receive sample on another board.
```

Comment:
```suggestion
This sample demonstrates how to use the LoRa radio driver to configure
the encoding settings and send data over the radio.

Transmitted messages can be received by building and flashing the accompanying
LoRa receive sample on another board.
```

As soon as you add (I hope you will!) the readme for the receive sample, they should both reference one another using `` :zephyr:code-sample:`...` ``

Reply:
I'll order another board tomorrow so I should be able to do it this weekend to add the receive README.